docs/contributing/release_guide.rst

```diff
@@ -4,6 +4,14 @@
 Release Guide
 *************
 
+.. contents:: Table of Contents
+   :depth: 1
+   :local:
+   :backlinks: none
+
+Introduction
+------------
```

Comment:
Using `---------` instead of `========` to match the style in the rest of this particular doc.